Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php

```diff
@@ -1,130 +1,363 @@
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
+        );
+
+        $handleFailedAuthorisationMethod = $this->getPrivateMethod(
+            AuthorisationWebhookHandler::class,
+            'handleFailedAuthorisation'
+        );
+
+        // Call the private method directly and provide required parameters
+        $result = $handleFailedAuthorisationMethod->invokeArgs($webhookHandler, [$this->orderMock, $this->notificationMock]);
+
+        // Assert the expected behavior based on the mocked logic and result
+        $this->assertInstanceOf(Order::class, $result);
+    }
+
+    public function testHandleAutoCapture()
+    {
+        // Set up expectations for the mocks
+        $this->orderMock->expects($this->any())
+            ->method('getPayment')
+            ->willReturn($this->orderMock);
+
+        $this->orderMock->expects($this->any())
+            ->method('getConfig')
+            ->willReturnSelf();
+
+        // Create an instance of AuthorisationWebhookHandler
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
+        );
+
+        // Use Reflection to access the private method
+        $handleAutoCaptureMethod = $this->getPrivateMethod(
+            AuthorisationWebhookHandler::class,
+            'handleAutoCapture'
+        );
+
+        // Call the private method directly and provide required parameters
+        $result = $handleAutoCaptureMethod->invokeArgs(
+            $webhookHandler,
+            [$this->orderMock, $this->notificationMock, true] // true indicates requireFraudManualReview
+        );
+
+        // Perform assertions on the result and expected behavior
+        $this->assertInstanceOf(Order::class, $result);
+    }
+
+    public function testHandleManualCapture()
+    {
+        // Set up expectations for handleManualCapture private method
+        $requireFraudManualReview = true; // Assuming this condition is met
+        $this->orderHelperMock->expects($this->never()) // Since the condition is true
+        ->method('setPrePaymentAuthorized');
+
+        $this->caseManagementMock->expects($this->once())
+            ->method('markCaseAsPendingReview')
+            ->with($this->orderMock, $this->notificationMock->getPspreference(), false);
+
+        // Create an instance of AuthorisationWebhookHandler
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            $this->orderHelperMock,
+            $this->caseManagementMock,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
```

Comment:
You can skip sending values for optional arguments. They are already null by default.